Untitled

```diff
@@ -0,0 +1,54 @@
+# SEED system status
+
+This page provides the following Information:
```

Comment:
I will suggest to keep this page out of this pr until we have real data in it.

Reply:
renamed the file.